invisible_cities/reco/xy_algorithm_test.py

```diff
@@ -217,44 +327,112 @@ def test_is_masked():
     assert not is_masked(sipm_alive , pos_masked)
 
 
-def test_masked_channels():
+
+def test_count_masked_all_active(datasipm_all_active):
+    xy0 = np.array([0, 0], dtype=np.float)
+    is_masked = datasipm_all_active.Active.values
+
+    # All sipms are active in run number 0
```

Comment:
At this moment, I can see that run number 0 has some sensor marked as not active. I think that you want to say "run number 1", here.

Reply:
Yes, this is a typo, It is run number 1